src/node_contextify.cc

```diff
@@ -956,27 +958,20 @@ class ContextifyScript : public BaseObject {
     bool timed_out = false;
     bool received_signal = false;
```

Comment:
I think in a previous version of this PR these were `volatile`… any reason why that changed?